css/templates/lune/page-vars.css

```diff
@@ -1,6 +1,7 @@
 :root {
   --content-max-width: 100%;
 
+  --z0: 0;
```

Comment:
same here - you wll not need all the z-indices that were already defined. (You will only need --z0)